backend/src/main/java/codesquad/airbnb/accomodation/domain/Provide.java

```diff
@@ -0,0 +1,26 @@
+package codesquad.airbnb.accomodation.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+@Getter
+@AllArgsConstructor
+@NoArgsConstructor // for @Entity
+public class Provide {
+
+    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    @Convert(converter = ProvideNameConverter.class)
+    private ProvideName name;
```

Comment:
CAPACITY, BED_ROOM 같은 타입을 저장하면 불필요한 코드를 줄일 수 있을것 같습니다

```suggestion
    @ Enumerated
    private ProvideType type;
```